crates/prql-compiler/tests/integration/dockerfiles/glaredb.Dockerfile

```diff
@@ -0,0 +1,12 @@
+# TODO: switch to `curlimages/curl`
+# glaredb binary releases are not compatible with alpine
+# So we can't use here now
+# https://github.com/GlareDB/glaredb/issues/1912
+FROM --platform=linux/amd64 docker.io/library/buildpack-deps:stable
+
+RUN <<EOF
+cd /usr/local/bin
+curl https://glaredb.com/install.sh | sh
+EOF
```

Comment:
I need to change this to

```
RUN cd /usr/local/bin && \
    curl https://glaredb.com/install.sh | sh
```

Otherwise the dockerfile wasn't parsing correctly for me.

Reply:
Thanks for pointing that.
It worked in my machine, I forgot to add `# syntax=docker/dockerfile:1.4` to the first line.